docs/content/reference/cli/authelia/authelia_storage_bans_user_list.md

```
@@ -0,0 +1,70 @@
+
+## authelia storage bans user list
+
+Lists user bans
+
+### Synopsis
+
+Lists user bans.
+
+This subcommand allows listing user bans from the regulation system.
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)